snow/engine/avalanche/state/unique_vertex.go

```diff
@@ -300,13 +300,13 @@ func (vtx *uniqueVertex) String() string {
 		len(txs),
 	))
 
-	parentFormat := fmt.Sprintf("\n    Parent[%s]: ID = %%s, Status = %%s",
+	parentFormat := fmt.Sprintf("\n    Parent[%s]: ID = %%s, Status = %%s", //nolint:perfsprint
```

Comment:
why are we disabling linting here? There were other places where we were able to replace  Sprintf with simpler alternative. What's the difference here?

Reply:
I tried changing it a few times but the code got more confusing. We don't stringify the unique vertex enough to sacrifice code clarity for performance IMO